recipes/create-dmg/all/conanfile.py

```diff
@@ -30,13 +30,15 @@ def build(self):
     def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
         self.copy("create-dmg", dst="bin", src=self._source_subfolder)
+        self.copy("*", dst=os.path.join("bin","support"), src=os.path.join(self._source_subfolder,"support"))
```

Comment:
Why in the bin folder. Upstream Makefile installs this folder in share folder.

Reply:
@MartinDelille configuration files and other needed files, are copied to `res` folder, which is covered by Conan already.

Reply:
Ok so if I understand correctly, I should replace `bin` by `res` here ?

Reply:
by `res/create_dmg`

Reply:
I fixed that here: https://github.com/conan-io/conan-center-index/pull/11569